src/leiningen/new/cryogen/themes/blue/html/post.html

```diff
@@ -15,6 +15,10 @@
     {% if disqus-shortname %}
     <div id="disqus_thread"></div>
     <script type="text/javascript">
+        var disqus_config = function () {
+              this.page.url = {{post.uri}};
```

Comment:
I think you need quotes around the values, e.g: `this.page.url = "{{post.url}}";